### ISSUE TITLE
aas_test_engines/test_cases/v3_0/api.py: `test_include_concept_descriptions` empty results error

### ISSUE THREAD

**Issue by Frosty2500** [admin-shell-io/aas-test-engines#78]
In `test_include_concept_descriptions` it is not checked if the data is empty, which leads to the error 
```console
    _assert('conceptDescriptions' in data, 'contains conceptDescriptions', Level.WARNING)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: argument of type 'NoneType' is not iterable

```
when testing our server.
This is fixed by adding the following assertion.
```python
_assert(data is not None, 'Result is not empty')
```
So when I use this method 
```python
def test_include_concept_descriptions(self):
    """
    Invoke with includeConceptDescriptions    """    request = generate_one_valid(self.operation, self.sample_cache, {'includeConceptDescriptions': True})
    data = _invoke_and_decode(request, self.conf, True)
    _assert(data is not None, 'Result is not empty')
    _assert('conceptDescriptions' in data, 'contains conceptDescriptions', Level.WARNING)
```
it is possible to test our server.

**Comment by mnmnk43434** [comment]
Hello, we tried to solve the issue.

This is what we did:

Added a check to ensure data is not None before asserting 'conceptDescriptions' in data

You can review changes in this commit: https://github.com/mnmnk43434/admin-shell-io-aas-test-engines-78/commit/8d3ae9828039476785a1f41cfbf523251cd53dda.


> [!CAUTION]
> Disclaimer: The concept of solution was created by AI and you should never copy paste this code before you check the correctness of generated code. Solution might not be complete, you should use this code as an inspiration only.

---

If you no longer want Latta AI to attempt solving issues on your repository, you can block this account.

**Comment by otto-ifak** [comment]
You are right, that is a bug. If the response of the server does not conform to the JSON schema, test execution is continued erroneously. This is fixed now.
